openassessment/xblock/openassessmentblock.py

```diff
@@ -586,7 +588,8 @@ def ora_blocks_listing_view(self, context=None):
             additional_css=["static/css/lib/backgrid/backgrid%s.css" % min_postfix],
             additional_js=["static/js/lib/backgrid/backgrid%s.js" % min_postfix],
             additional_js_context={
-                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled
+                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled,
+                "ORA_GRADING_MICROFRONTEND_URL": settings.ORA_GRADING_MICROFRONTEND_URL
```

Comment:
Do we need handling for when this config isn't present?

Reply:
Yeah, I'd agree and say this should maybe be`getattr(settings, 'ORA_GRADING_MICROFRONTEND_URL', '')`

Reply:
Just to check I threw together this PR(which includes an old commit too, my bad) and pylint doesn't complain. Are you sure we needed that ignore? https://github.com/openedx/edx-ora2/pull/1797